### PR TITLE
 	Updates in packaging

### PR DESCRIPTION
**Pull request by sergiomb2**
Patch1 
 - Fix mix tabs and spaces with vim :retab

Patch2 
- unbundle several jars not all
- ant just linux builds
- unzip dist/*.zip as files to install, is all we want to install
- use own png for .desktop and use own startup script just do a symbol link to
  /usr/bin it works superb
- we have 64 bits, 32 bits and generic builds, generic build seems to be noarch
  build, but others are arched ... , TODO mark package as noarch for generic build.
  Also as side effect of arched builds we need add %global debug_package %{nil},
  because build ends with Empty %files file
  /builddir/build/BUILD/dbgl-0.80/debugfiles.list

Patch3 
- Use external .desktop file easier to send to upstream
- Add Packaging:AppData

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+mock_travis:
+  mock_config: fedora-25-x86_64
+  packages_buildrequires_git: UnitedRPMs/dbgl
```
Comment (igor-raits) [inline .travis.yml:3]: sorry, but no.

File: dbgl.spec [hunk 2]
```diff
@@ -1,30 +1,41 @@
...
+Requires:       dosbox >= 0.70
+Requires:       eclipse-swt
+Requires:       hsqldb
+Requires:       java >= 1:1.7.0
+Requires:       java-headless >= 1:1.7.0
+Requires:       jpackage-utils
+Requires:       SDL_net
+Requires:       SDL_sound
+Source0:        http://members.quicknet.nl/blankendaalr/dbgl/download/src080.zip
+Source1:        https://raw.githubusercontent.com/UnitedRPMs/dbgl/master/dbgl.desktop
```
Comment (igor-raits) [inline dbgl.spec:37]: no.

**Review by igor-raits — CHANGES_REQUESTED** [review]
though in general looks good, but too many UnitedRPMs bullshit. Remove it and we're in business

**Comment by sergiomb2** [comment]
OK, you may merge just the first 3 commits .

File: dbgl.spec [hunk 3]
```diff
@@ -1,30 +1,41 @@
...
+Source2:        https://raw.githubusercontent.com/UnitedRPMs/dbgl/master/dbgl.appdata.xml
```
Comment (xvitaly) [inline dbgl.spec:38]: Are you serious? Trying to fetch files from this shit repo?

Comment (Tigro) [reply dbgl.spec:38]: Yes, these files should be in our git as at this time.

Comment (sergiomb2) [reply dbgl.spec:38]: it is a way that @paulcarroty found to build rpm in travis , I haven't look at it yet ...

Comment (xvitaly) [reply dbgl.spec:38]: Upload this files to your pull request and change URLs to https://raw.githubusercontent.com/RussianFedora/dbgl/.

Comment (sergiomb2) [reply dbgl.spec:38]: (again) you may merge just the first 3 commits and ignore the travis stuff (the others commits) ...

Comment (paulcarroty) [reply dbgl.spec:38]: @sergiomb2 Russians are always full of bullshit, some things never changes.

Comment (igor-raits) [reply dbgl.spec:38]: I have to note that @paulcarroty is also Russian (well, AFAIK Ukrainian)

Comment (xvitaly) [reply dbgl.spec:38]: @paulcarroty You still removing copyrights from SPEC files taken from RussianFedora and RPMFusion?

Comment (paulcarroty) [reply dbgl.spec:38]: @xvitaly proofs or shut up.

@ignatenkobrain https://en.wikipedia.org/wiki/Continuous_integration

Comment (xvitaly) [reply dbgl.spec:38]: @paulcarroty http://i.imgur.com/aju3LVO.png

Comment (paulcarroty) [reply dbgl.spec:38]: @xvitaly 
My latest major commit:
```
curl -s  https://github.com/UnitedRPMs/chromium-freeworld/blob/c4941b4a3851d562bf2311f4df392e874fe28462/chromium.spec | grep -i russianfedora | wc -l
132
``` 
Someone lying as Russia Tuday.

Comment (xvitaly) [reply dbgl.spec:38]: @paulcarroty Nice try. Let's just see Git history: http://copr-dist-git.fedorainfracloud.org/cgit/paulcarroty/test/chromium.git/commit/?h=f24&id=1c4e6fc3ccd9fa2f32bae4130356ff23dba6f35c

You deleted your COPR, but not Git repositories. :-)

Comment (Vascom) [reply dbgl.spec:38]: @paulcarroty Пожалуйста, не вмешивайся. Ты сам не хотел иметь ни чего общего с russianfedora.

**Comment by sergiomb2** [comment]
Sorry this PR, that I promoted, wasn't a good idea, I wasn't expect that spec change to build in .travis builder , I just want improve packaging of one java package. So I will fork this project again and send the same commits but from my github home , if we got another improvements  , I will forward from my repo to unitedrpms repo. 
About dbgl I was looking for a solution for a gui to dosbox some time ago , yesterday I found this .spec and didn't notice that we have a fork for RussianFedora , I spent all night improving the package . Today I though in publish my work , and though where I can build it public ? , since I don't know if it is allowed in Fedora or RPMFusion , I excluded them , so I choose go directly to UnitedRpms , which was made for big hacking ! we got dbgl-0.80-2.fc25.x86_64.rpm built here : 
https://github.com/UnitedRPMs/travis-rpm-builder/releases/tag/dbglFEDORA-25-X86_64201612201212

So closing the PR , please wait for new one , Thanks.

**Pull request closed without merging**